eclass/verify-sig.eclass

```diff
@@ -141,6 +141,9 @@ verify-sig_verify_detached() {
 	einfo "Verifying ${filename} ..."
 	case ${VERIFY_SIG_METHOD} in
 		openpgp)
+			# gpg can't handle very long TMPDIR
+			# https://bugs.gentoo.org/854492
+			local -x TMPDIR=/tmp
```

Comment:
Are we sure we want /tmp directly and not an `mktemp` dir? That would head of certain classes of vulnerabilities, I guess, but I've not investigated the potential in depth.

Reply:
gemato does use a `tempfile.TemporaryDirectory()` on top of that.